setup.cfg

```diff
@@ -2,7 +2,51 @@
 universal = 1
 
 [metadata]
+name = pycodestyle
+author = Johann C. Rocholl
+author_email = johann@rocholl.net
+maintainer = Ian Lee
+maintainer_email = IanLee1521@gmail.com
+license = Expat
```

Comment:
I know it's not a regression, but as far as I can tell the values for this field should be spdx values, of wich "MIT" would be the appropriate value here